Find a functionally equivalent code:android.net.ConnectivityManager.isNetworkTypeMobile:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * Checks if a given type uses the cellular data connection.
 * This should be replaced in the future by a network property.
 * @param networkType the type to check
 * @return a boolean - {@code true} if uses cellular network, else {@code false}
 * {@hide}
 */

@@ -9,14 +9,15 @@
         case TYPE_MOBILE_IMS:
         case TYPE_MOBILE_CBS:
         case TYPE_MOBILE_IA:
+        case TYPE_MOBILE_EMERGENCY:
             return true;
         default:
             return false;

